adal/src/androidTest/java/com/microsoft/aad/adal/AndroidTestHelper.java

```diff
@@ -56,8 +57,9 @@ protected void setUp() throws Exception {
         System.setProperty("dexmaker.dexcache", getInstrumentation().getTargetContext().getCacheDir().getPath());
 
         // ADAL is set to this signature for now
```

Comment:
We're still setting signatures. Instead of using the hardcoded test app name, we're using the auto-generated app.